tests/test_feature_extraction_common.py

```diff
@@ -18,6 +18,57 @@
 import os
 import tempfile
 
+from transformers.file_utils import is_torch_available, is_vision_available
+
+
+if is_torch_available():
+    import numpy as np
+    import torch
+
+if is_vision_available():
+    from PIL import Image
+
+
+def prepare_inputs(feature_extract_tester, equal_resolution=False, numpify=False, torchify=False):
```

Comment:
Can we maybe rename it to `prepare_inputs_for_image_models`? or something that includes `image`?

Reply:
Renamed to `prepare_image_inputs`.